DesktopModules/Vanjaro/Core/Library/Factories/NotificationTaskFactory.cs

```diff
@@ -19,7 +19,7 @@ internal static int GetNotificationCount(int PortalID)
                 if (NotificationTasks == null)
                 {
                     List<INotificationTask> ServiceInterfaceAssemblies = new List<INotificationTask>();
-                    string[] binAssemblies = Directory.GetFiles(Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "bin")).Where(c => c.EndsWith(".dll") && (c.Contains("Vanjaro.UXManager.Extensions") || c.Contains("Vanjaro.Core.Extensions"))).ToArray();
+                    string[] binAssemblies = Directory.GetFiles(Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "bin")).Where(c => c.EndsWith(".dll") && (c.Contains("Vanjaro.UXManager.Extensions") || c.Contains("Vanjaro.Core.Extensions") || c.Contains("Mandeeps.DNN.Modules.LiveBlog"))).ToArray();
```

Comment:
This doesn't make sense. Why hardcode Vanjaro Extensions or LiveBlog. This needs to be fixed.